src/views/includes/macros.njk

```diff
@@ -51,11 +51,20 @@
   {{- capitalizedWords.join(' ') -}}
 {%- endmacro -%}
 
+// cannot get the values from the locales.
+{% set capitalContributionSubTypesMap = {
+  "MONEY": i18n.capitalContribution.contributionSubTypes.money,
```

Comment:
locales fail - tried placing this here and in the check-your-aswers-template both at the top and within the lp scope.

Reply:
See above